pkg/products/fuse/reconciler.go

```diff
@@ -237,6 +211,58 @@ func (r *Reconciler) createResource(ctx context.Context, resourceName string, se
 	return resource, nil
 }
 
+func (r *Reconciler) scaleDown(ctx context.Context, serverClient k8sclient.Client) {
+	var replicas int32 = 0
+
+	r.log.Info("Scaling down fuse deployments")
+
+	var deployment = &k8sappsv1.Deployment{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "syndesis-operator",
+			Namespace: r.Config.GetOperatorNamespace(),
+		},
+	}
+
+	_, err := controllerutil.CreateOrUpdate(ctx, serverClient, deployment, func() error {
+		deployment.Spec.Replicas = &replicas
+		return nil
+	})
+	if (k8serr.IsNotFound(err)) {
+		r.log.Info("syndesis-operator deployment not found")
+	} else if (err != nil) {
+		r.log.Error("Error scaling syndesis-operator down", err)
+	}
+
+	// name/namespace to scale down
+	dcs := []string{
+		"broker-amq",
+		"syndesis-meta",
+		"syndesis-oauthproxy",
+		"syndesis-prometheus",
+		"syndesis-server",
+		"syndesis-ui",
+	}
+
+	for _, deployment := range dcs {
+
+		var dc = &appsv1.DeploymentConfig{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      deployment,
+				Namespace: r.Config.GetNamespace(),
+			},
+		}
+		_, err := controllerutil.CreateOrUpdate(ctx, serverClient, dc, func() error {
+			dc.Spec.Replicas = replicas
+			return nil
+		})
+		if (k8serr.IsNotFound(err)) {
+			r.log.Infof("deployment not found", l.Fields{"deployment":dc.Name})
+		} else if (err != nil) {
```

Comment:
Hey, do we not want to set Fuse in progress in case of failure in updating DCs, Deployment or removing alerts? I guess because the alerts have bsen updated to fire on presence of the pod we can assume all went well if alerts don't fire, this is more of a question rather than anything else.